flutter/lib/src/flutter_enricher.dart

```diff
@@ -0,0 +1,168 @@
+    // if it's missing
+    // ignore: invalid_null_aware_operator
+    final languageTag = _window.locale?.toLanguageTag();
+
+    // The editor says it's fine without a `?` but the compiler complains
+    // if it's missing
+    final availableLocales =
+        // ignore: invalid_null_aware_operator
+        _window.locales?.map((it) => it.toLanguageTag()).toList();
+
+    return <String, dynamic>{
+      'is_24_hour_format': _window.alwaysUse24HourFormat,
+      if (languageTag != null) 'locale': languageTag,
+      if (availableLocales != null) 'available_locales': availableLocales,
+      'timezone': DateTime.now().timeZoneName,
+    };
+  }
+
+  Map<String, dynamic> _getFlutterContext() {
+    final currentLifecycle = _widgetsBinding.lifecycleState;
+
+    return <String, dynamic>{
+      if (debugBrightnessOverride != null)
+        'debug_brightness_override': debugBrightnessOverride,
+      if (debugDefaultTargetPlatformOverride != null)
+        'debug_default_target_platform_override':
+            debugDefaultTargetPlatformOverride,
+      if (_window.initialLifecycleState.isNotEmpty)
+        'initial_lifecycle_state': _window.initialLifecycleState,
+      if (_window.defaultRouteName.isNotEmpty)
+        'default_route_name': _window.defaultRouteName,
+      if (currentLifecycle != null)
+        'current_lifecycle_state': describeEnum(currentLifecycle),
+      // Seems to always return false.
+      // Also always fails in tests.
+      // 'window_is_visible': _window.viewConfiguration.visible,
+    };
+  }
+
+  Map<String, dynamic> _getAccessibilityContext() {
+    return <String, dynamic>{
+      'accessible_navigation':
+          _window.accessibilityFeatures.accessibleNavigation,
+      'bold_text': _window.accessibilityFeatures.boldText,
+      'disable_animations': _window.accessibilityFeatures.disableAnimations,
+      'high_contrast': _window.accessibilityFeatures.highContrast,
+      'invert_colors': _window.accessibilityFeatures.invertColors,
+      'reduce_motion': _window.accessibilityFeatures.reduceMotion,
+    };
+  }
+
+  SentryDevice _getDevice(SentryDevice? device) {
+    final orientation = _window.physicalSize.width > _window.physicalSize.height
+        ? SentryOrientation.landscape
+        : SentryOrientation.portrait;
+
+    return (device ?? SentryDevice()).copyWith(
+      orientation: orientation,
+      screenHeightPixels: _window.physicalSize.height.toInt(),
+      screenWidthPixels: _window.physicalSize.width.toInt(),
+      screenDensity: _window.devicePixelRatio,
+      theme: describeEnum(_window.platformBrightness),
+    );
+  }
+
+  List<SentryRuntime> _getRuntimes(List<SentryRuntime>? runtimes) {
+    var flutterRuntimeDescription = '';
+
+    // See
+    // - https://flutter.dev/docs/testing/build-modes
+    // - https://github.com/flutter/flutter/wiki/Flutter%27s-modes
+    if (_checker.isWeb) {
+      if (_checker.isDebugMode()) {
+        flutterRuntimeDescription = 'Flutter with dartdevc';
```

Comment:
Do we want to add a `compiler` (or something along those lines) field to `Runtime`?  It could then hold the information which compiler (AOT, VM, dart2js, dartdevc) was used.

Reply:
sounds useful